test/docfx.RegressionTest/Options.cs

```diff
@@ -7,6 +7,9 @@ namespace Microsoft.Docs.Build
 {
     internal class Options
     {
+        [Option("warm-up")]
+        public bool WarmUp { get; set; } = false;
```

Comment:
nit: move it after `[Value]` before other options.

Reply:
Fixed.